src/models/MemberPublicKey.model.ts

```diff
@@ -0,0 +1,12 @@
+import { prop } from '@typegoose/typegoose';
+
+export class MemberPublicKeyModelSchema {
```

Comment:
This isn't storing the creation date as required in #7. This is important for auditing purposes.